src/lib.rs

```diff
@@ -83,10 +88,15 @@ mod atc;
 mod bcn;
 mod etc;
 mod pvrtc;
+mod crunch;
```

Comment:
As crunch uses alloc,
it should be locked behind a `#[cfg(feature = "alloc")]` to allow complete no-std deployments.